app/routers/weight.py

```diff
@@ -0,0 +1,64 @@
+
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.responses import RedirectResponse
+
+from app.database.models import User
+from app.dependencies import get_db
+from app.dependencies import templates
+
+
+router = APIRouter(tags=["weight"],)
+
+
+@router.get("/weight")
+async def weight_form(
+        request: Request,
+        session: Session = Depends(get_db),
+        target: float = None,
```

Comment:
if it defaults to none you should do
`Union[float, None]`

Reply:
Done